### PR TITLE
adds a monstrosity into the game

### PR DESCRIPTION
**Pull request by c420-o**
## About The Pull Requesthorror

hell
Oh god, what the fuck is this thing? This is a crime against nature. We can't let this thing live. You can't even kill it? God save us.

## Why It's Good For The Game horror

It's not good for the game. At all. Nothing about this is good. You'd be better off adding more trading cards. 

## Changelog
:cl:
add: Adds a horrible monster to the game.

/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
They are already in the game bro

**Comment by Fikou** [comment]
?? this is already in the game

File: code/modules/mob/living/simple_animal/monstrosity.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/mob/living/simple_animal/monstrosity
+	name = "...what?"
+	desc = "WHAT IT IS THAT!? WHAT THE FUCK IS THAT!?"
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/monstrosity.dm:3]: IT IS THAT

**Comment by maxymax13** [comment]
if i had a dime for every time this thing has been pred as an addition i'd have 2 dimes. which is not a lot but it's weird that it's happened twice

**Comment by notfrying1pans** [comment]
this mob's already in the game in a space ruin you can find

**Pull request closed without merging**